kvmagent/kvmagent/test/shareblock_testsuite/test_sharedblock_batch_get_volume_size.py

```
@@ -37,7 +37,7 @@ def test_sharedblock_batch_get_volume_size(self):
         self.assertEqual(rsp.success, True, "iscsiadm login failed")
 
         # test connect shareblock
-        r, o = bash.bash_ro("ls /dev/disk/by-id | grep scsi|awk -F '-' '{print $2}'")
+        r, o = bash.bash_ro("ls /dev/disk/by-id | grep scsi-3 | awk -F '-' '{print $2}'")
```

Ruff (0.15.2)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)